README.md

```diff
@@ -1 +1,109 @@
-# LunarBase
\ No newline at end of file
+# LunarBase
+
+This is the LunaTechs base RPG Maker plugin development project. Its used to
+demonstrate and show you our basic workflow with [Haxe] and [LunaTea].
```

Comment:
Missing links for Haxe and LunaTea? The usage of [] make it look like it.

Reply:
Good catch, will update and push changes.